middleware/build.gradle

```diff
@@ -15,15 +26,23 @@ dependencies {
     testImplementation libraries.assertJ
     testImplementation libraries.junit
     testImplementation libraries.guava
-}
-
-buildscript {
-    tasks.withType(JavaCompile) {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
-    }
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
```

Comment:
can we do `implementation` since `compile` will be removed soon?

Reply:
Also doesn't this miss the version part of the dependency?

Reply:
can do on the implementation.  version of kotlin is specified higher up and will error if specified here....maybe i'm dong something wrong...lmk

Reply:
+1 to what @ramonaharrison asked :-)

Reply:
done! was wrong about the version label, i added that in places where it was missing! great catch!